style.css

```diff
@@ -17,7 +17,7 @@ h1{
 
 }
 
-header{
+.nav-header{
     display: flex;
 
```

Comment:
remove space

Reply:
removed extra space from style css and also used color variables